qiskit/transpiler/preset_passmanagers/level1.py

```diff
@@ -307,7 +307,17 @@ def _opt_control(property_set):
             and pass_manager_config.layout_method is None
         ):
 
-            def _post_layout_condition(property_set):
+            def _run_post_layout_condition(property_set):
+                if _trivial_not_perfect(property_set):
+                    vf2_stop_reason = property_set["VF2Layout_stop_reason"]
+                    if (
+                        vf2_stop_reason is None
+                        or vf2_stop_reason != VF2LayoutStopReason.SOLUTION_FOUND
+                    ):
+                        return True
+                return False
+
+            def _apply_post_layout_condition(property_set):
```

Comment:
The comment on the lines below looks to be outdated. Can you update it?

Reply:
Updated in: https://github.com/Qiskit/qiskit-terra/pull/7991/commits/96e4d281ad2feb75cd1844e82349c195342f42b3